requirements-test.txt

```diff
@@ -17,7 +17,6 @@ requests
 ray[adag]==2.35
 sentence-transformers # required for embedding
 soundfile # required for audio test
-compressed-tensors==0.4.0 # required for compressed-tensors
```

Comment:
Do we need to add the dependency in `requirements-test` if it's already in `-common`? Seems like violating SSOT otherwise

Reply:
I removed the dep in `requirements-test` to move it to `requirements-common` - did you miss that?

Reply:
Yeah sorry got confused when comparing diffs with [the previous commit](https://github.com/vllm-project/vllm/pull/7277/files)

Reply:
However, seems like some CI tests only use requirements-test, see https://github.com/vllm-project/vllm/pull/7277/files#r1711531709

Reply:
I can't find any references to code using requirements-test.txt directly https://github.com/search?q=repo%3Avllm-project%2Fvllm%20%22requirements-test%22&type=code

Reply:
The CI tests may have changed since my previous PR